app/code/Magento/Multishipping/Test/Mftf/Test/StoreFrontCheckVatIdAtAccountCreateWithMultishipmentTest.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StoreFrontCheckVatIdAtAccountCreateWithMultishipmentTest">
+        <annotations>
+            <features value="Multishipment"/>
```

Comment:
Isn't it possible to verify this change without the multishipment? Ideally, I believe this test should be on the customer module.

Reply:
Has been moved to the customer module.

Reply:
Moved back as discussed.